db/migrate/20250113222716_remove_state_file_archived_intake_requests_fields.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveStateFileArchivedIntakeRequestsFields < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured do
+      remove_column :state_file_archived_intake_access_logs, :ip_address
+      remove_column :state_file_archived_intake_requests, :details
```

Comment:
this is silly but since the migration that adds this column is also on this PR, couldn't you just not add it?